spec/system/user_page/user_preferences_profile_spec.rb

```diff
@@ -20,72 +20,94 @@
   end
 
   describe "enforcing required fields" do
-    before do
-      UserRequiredFieldsVersion.create!
-      UserField.create!(
-        field_type: "text",
-        name: "Favourite Pokemon",
-        description: "Hint: It's Mudkip.",
-        requirement: :for_all_users,
-        editable: true,
-      )
-    end
+    context "when there's a field required for all users" do
```

Comment:
The entire first context remains unmodified, but has been indented by one level.